core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -84,6 +84,11 @@ sealed abstract class IO[+A] private () extends IOPlatform[A] {
 
   def evalOn(ec: ExecutionContext): IO[A] = IO.EvalOn(this, ec)
 
+  def startOn(ec: ExecutionContext): IO[FiberIO[A @uncheckedVariance]] = start.evalOn(ec)
```

Comment:
This would probably need a similar change to what Daniel mentioned below:

```suggestion
  def startOn(ec: ExecutionContext): IO[FiberIO[A @uncheckedVariance]] = evalOn(ec).start
```

Reply:
ignore ;)